.github/CODEOWNERS

```diff
@@ -22,7 +22,7 @@ content-modules/opentelemetry-proto      @open-telemetry/docs-maintainers @open-
 content-modules/opentelemetry-specification  @open-telemetry/docs-maintainers @open-telemetry/specs-approvers
 content-modules/semantic-conventions     @open-telemetry/docs-maintainers @open-telemetry/specs-semconv-approvers
 content/en/blog/                         @open-telemetry/docs-maintainers
-content/en/community/end-user/           @open-telemetry/docs-approvers @open-telemetry/end-user-wg
+content/en/community/end-user/           @open-telemetry/docs-approvers @open-telemetry/sig-end-user-approvers
```

Comment:
Should this be `sig-end-user-maintainers`? The end-user SIG only has maintainers at the moment.

Reply:
As far as I understand it maintainers are a subteam of the approvers, so if the approvers are tagged the maintainers will be tagged as well. We do this for consistency, but we can have the maintainers here if that's the preferred group

Reply:
Good point. I keep forgetting those are sub-teams!

Reply:
GitHub doesn't seen to think that this team exists.

Reply:
ah, we need to fix repo permissions as well